modules/querier/config.go

```diff
@@ -55,6 +55,10 @@ type MetricsConfig struct {
 	// between 0.0 and 1.0.  If a block overlaps the time window by less than this value,
 	// then we skip the columns. A value of 1.0 will always load the columns, and 0.0 never.
 	TimeOverlapCutoff float64 `yaml:"time_overlap_cutoff,omitempty"`
+
+	// ExemplarSampling is the percentage of exemplars to sample.
+	// The value must be between 0.0 and 1.0.
+	ExemplarSampling float64 `yaml:"exemplar_sampling,omitempty"`
```

Comment:
I lean towards configuring a max number of exemplars per query, or per step (data point).  Something that scales with the response, and not the tracing volume.   If this is percent of spans, it will be hard to set a value here that works for 1K spans/s and 1M span/s at the same time.  Sorry if I am misunderstanding how the sampling is occurring, haven't reviewed thoroughly.

Reply:
This is a leftover from a past implementation. Now it's configured to max 100 and 2 per bucket.

Removing!